core/conversion/converters/impl/activation.cpp

```diff
@@ -75,9 +77,10 @@ auto acthardtanh TORCHTRT_UNUSED =
                     new_layer->setBeta(max);
 
                     new_layer->setName(util::node_info(n).c_str());
+                    new_layer->setOutputType(0, in->getType());
```

Comment:
What would cause this to not be the case? would this interfere with TensorRT's mixed precision behavior?